sechub-administration/src/main/java/com/mercedesbenz/sechub/domain/administration/mapping/UpdateMappingService.java

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 package com.mercedesbenz.sechub.domain.administration.mapping;
 
-import static com.mercedesbenz.sechub.sharedkernel.validation.AssertValidResult.*;
+import static com.mercedesbenz.sechub.sharedkernel.validation.AssertValidResult.assertValid;
```

Comment:
Normally our code setup for imports says: static imports always * - here okay.

Maybe we should implement a spotless rule for this.